doc/DEMO_TESTNET.md

```diff
@@ -173,127 +204,174 @@ cp contrib/config.toml coordinatord_config.toml
 ```
 
 Make sure that the `postgres_uri` is `postgresql://revault:revault@localhost:5432/coordinator_db`.
-change the noise keys with the public noise keys of the participants.
 
-Your config file now should look like this (with your keys in it instead of those dummy keys, obviously):
+Also, update the `data_dir` to `./coordinatord_data`.
+
+Your config file now should look like this:
 ```
 daemon=true
-data_dir = "./revault_coordinatord"
+data_dir = "./coordinatord_data"
 log_level = "debug"
 
 postgres_uri = "postgresql://revault:revault@localhost:5432/coordinator_db"
 
-managers = [
-    "d2deeb8398f47789e1f5118c42834031e3722817a432192b74c363fdb36cc634",
-]
+managers = []
 
-stakeholders = [
-    "eecd2a93f5b09b88519f38d620aa127333d1934987d18001132f07ffa3596c65",
-    "614ad96890c309b8da6915ddb9eb1135caf228120178833e70798f20e0783b16",
-]
+stakeholders = []
 
 watchtowers = []
 ```
+We'll update the coordinator configuration later, when we'll have the
+noise public keys of the participants.
 
-start the `coordinatord`:
+For now, let's just retrieve the coordinator's noise public key - start the `coordinatord`:
 
 ```
-~/revault_tutorial/coordinatord$
 cargo run -- --config coordinatord_config.toml
 ```
 
-Then please keep note of the noise key printed at startup, it will ask
-by the install screen of each participants as the coordinator public
-noise key.
+Then keep note of the noise key printed at startup - the revault installer
+will ask for it soon.
+
+Make sure that users not in your network can reach the coordinator.
+[ngrok](https://ngrok.com/) may be of help :)
 
-In order to permit other user to send request to the coordinator you can
-give them a [ngrok](https://ngrok.com/) endpoint `ngrok tcp 8383`.
+Now kill the cosigner, we don't need it for now.
 
-## Getting started as a stakeholder.
+## Getting started as a stakeholder
 
-### Setting up the cosigner.
+### Setting up the cosigner
```

Comment:
the *Cosigning Servers*

I think it may be confusing otherwise as it's the same terminology as classical multisig